src/app/components/client/toolbar/AppPicker.tsx

```diff
@@ -0,0 +1,373 @@
+          action: "bento-app-link-click",
+          label: productData.gaLabel,
+        });
+      }
+    });
+    if (itemKey === "mozilla") {
+      mozillaLinkRef.current?.click();
+      gaEvent({
+        category: "bento",
+        action: "bento-app-link-click",
+        label: "Mozilla",
+      });
+    }
+  };
+
+  return (
+    <AppPickerTrigger
+      label={l10n.getString("toolbar-app-picker-trigger-title")}
+      onAction={onSelect}
+      referringHost={referringHost}
+    >
+      <Item
+        key={products.vpn.id}
+        textValue={l10n.getString("toolbar-app-picker-product-vpn")}
+      >
+        <a
+          ref={linkRefs.vpn}
+          href={products.vpn.url}
+          className={`${styles["menuLink"]} ${styles.vpnLink}`}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <Image src={VpnLogo} alt="" width={16} height={16} />
+          {l10n.getString("toolbar-app-picker-product-vpn")}
+        </a>
+      </Item>
+      <Item
+        key={products.relay.id}
+        textValue={l10n.getString("toolbar-app-picker-product-relay")}
+      >
+        <a
+          ref={linkRefs.relay}
+          href={products.relay.url}
+          className={`${styles["menuLink"]} ${styles.relayLink}`}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <Image src={RelayLogo} alt="" width={16} height={16} />
+          {l10n.getString("toolbar-app-picker-product-relay")}
+        </a>
+      </Item>
+      <Item
+        key={products.pocket.id}
+        textValue={l10n.getString("toolbar-app-picker-product-pocket")}
+      >
+        <a
+          ref={linkRefs.pocket}
+          href={products.pocket.url}
+          className={`${styles["menuLink"]} ${styles.pocketLink}`}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <Image src={PocketLogo} alt="" width={16} height={16} />
+          {l10n.getString("toolbar-app-picker-product-pocket")}
+        </a>
+      </Item>
+      <Item
+        key={products.fxDesktop.id}
+        textValue={l10n.getString("toolbar-app-picker-product-fx-desktop")}
+      >
+        <a
+          ref={linkRefs.fxDesktop}
+          href={products.fxDesktop.url}
+          className={`${styles["menuLink"]} ${styles.fxDesktopLink}`}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <Image src={FxDesktopLogo} alt="" width={16} height={16} />
+          {l10n.getString("toolbar-app-picker-product-fx-desktop")}
+        </a>
+      </Item>
+      <Item
+        key={products.fxMobile.id}
+        textValue={l10n.getString("toolbar-app-picker-product-fx-mobile")}
+      >
+        <a
+          ref={linkRefs.fxMobile}
+          href={products.fxMobile.url}
+          className={`${styles["menuLink"]} ${styles.fxMobileLink}`}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <Image src={FxMobileLogo} alt="" width={16} height={16} />
```

Comment:
AH! I think the forced width/height of these images is causing some mighty big skew:

INPUT: 
![](https://raw.githubusercontent.com/mozilla/blurts-server/d6fb6bd14bdce089011594546b05635f687ffcef/src/app/components/client/toolbar/images/fx-mobile.png)

OUTPUT:
![](https://user-images.githubusercontent.com/4251/248320644-baac0126-7af6-429c-8faf-a8128f321fa8.png)

Reply:
Oh good point, that one is not rectangular. I don't know where it came from, so I can't find an SVG version, but here it at least is in the right proportions: c1b04c6b9